src/views/DetailedView/BookingSystem/ToggleButtons/index.tsx

```diff
@@ -4,16 +4,36 @@ import {
   RightToggleButton,
   StyledButtonGroup,
 } from "./toggleButtons.styled.tsx";
+import { DatesValues } from "../useCalendarDates.tsx";
+import { FunctionComponent } from "react";
 
-export const ToggleButtons = () => {
-  return (
-    <StyledButtonGroup>
-      <LeftToggleButton value="starts">
-        <Typography variant="h5">starts at</Typography>
-      </LeftToggleButton>
-      <RightToggleButton value="ends">
-        <Typography variant="h5">ends at</Typography>
-      </RightToggleButton>
-    </StyledButtonGroup>
-  );
+interface Props {
+  value: DatesValues;
+}
+
+export const ToggleButtons: FunctionComponent<Props> = ({ value }) => {
+  if (!value) {
+    return null;
+  }
+
+  const getDateString = (date: Date | null) => {
+    return `${date?.getDate()}.${
+      (date?.getMonth() ?? 0) + 1
+    }.${date?.getFullYear()}`;
+  };
+
+  if (Array.isArray(value)) {
```

Comment:
You need to return something from this component, but you are not doing that if value is not an array